README.md

```diff
@@ -11,6 +11,10 @@ Docblock Annotations Parser library (extracted from [Doctrine Common](https://gi
 
 See the [doctrine-project website](https://www.doctrine-project.org/projects/doctrine-annotations/en/latest/index.html).
 
+## Contributing
+
+Please always add native types where possible. [Coding Standard Guidelines.](https://www.doctrine-project.org/projects/doctrine-coding-standard/en/latest/reference/index.html#introduction)
```

Comment:
I don't think we should repeat the coding standard, IMO we should just point to it: 
```suggestion
When making a pull request, make sure your changes follow the
[Coding Standard Guidelines](https://www.doctrine-project.org/projects/doctrine-coding-standard/en/latest/reference/index.html#introduction).